cordova-lib/src/plugman/registry/registry.js

```diff
@@ -364,6 +364,28 @@ function fetchPlugin(plugin, client, useNpmRegistry) {
     });
 }
 
+function processPluginVersion(plugin) {
+    // If plugin includes a version that is a caret range, the ancient version of npm we're using won't now how to
+    // handle it. So we'll use our current version of semver to turn it into a usable range.
+
+    var parts = plugin.split('@');
+    var version = parts[1];
+
+    if (!version || version.charAt(0) !== '^') {
+        return plugin;
+    }
+
+    var validRange = semver.validRange(version, /* loose */ true);
+    if (!validRange) {
+        return plugin;
+    }
+
+    console.log('Returning processed plugin range: ' + parts[0] + '@"' + validRange + '"');
```

Comment:
Grrr... That was actually debug code I intended to remove. Thought I caught them all.